src/main/java/core/basesyntax/service/FruitShopServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.TransactionDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.filereader.FileParser;
+import core.basesyntax.service.filereader.FileReader;
+import core.basesyntax.service.filewriter.FileWriter;
+import core.basesyntax.service.interfaces.FruitShopService;
+import core.basesyntax.service.interfaces.strategy.TransactionStrategy;
+import core.basesyntax.service.transactions.BalanceTransactionHandler;
+import core.basesyntax.service.transactions.PurchaseTransactionHandler;
+import core.basesyntax.service.transactions.ReturnTransactionHandler;
+import core.basesyntax.service.transactions.SupplyTransactionHandler;
+import java.util.List;
+
+public class FruitShopServiceImpl implements FruitShopService {
+    private final TransactionDao transactionDao;
+    private final FileReader readFromFile;
+    private final FileParser parseDataFromFile;
+    private final FileWriter writeToFile;
```

Comment:
you changed class names but forgot to change variable names. Should be Nouns, not verbs